server/src/User.js

```diff
@@ -0,0 +1,14 @@
+import mongoose from 'mongoose';
+
+const userSchema = new mongoose.Schema({
+  id: String,
+  twitterProfile: {
+    type: Map,
+    of: Object,
+  },
+});
+
+// Hack to prevent us from initializing the model multiple times. See
+// https://github.com/dherault/serverless-offline/issues/258.
```

Comment:
I'm not going to click this because I don't want to know.

Reply:
I actually copy/pasted this from the Bit model, I assume it's the same here though